srcpkgs/nushell/template

```diff
@@ -11,8 +11,8 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="MIT"
 homepage="https://www.nushell.sh/"
 changelog="https://www.nushell.sh/blog/"
-distfiles="https://github.com/nushell/nushell/archive/${version}.tar.gz"
-checksum=4dcbe38b35902a98df3f213fef5f69dcd9870975a18e2a7feb002534e68d848b
+distfiles="https://github.com/nushell/${pkgname}/archive/refs/tags/${version}.tar.gz"
```

Comment:
Please keep `nushell` spelled out

Reply:
Will fix in the next push. Is this guideline something for all packages or just this one?